app/partials/project-card.cjsx

```diff
@@ -12,6 +12,9 @@ counterpart.registerTranslations 'en',
 module.exports = React.createClass
   displayName: 'ProjectCard'
 
+  projectOwner: ->
+    apiClient.type(@props.project.links.owner.type).get(@props.project.links.owner.id)
```

Comment:
I was thinking about having the client cache a subject by what it's constructed URL would be. So instead of looking to see if its ID already existed in the cache it would instead look at what URL it was about request and if it had already requested that.